src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The nth-child selectors are not correctly activating the intended number of stars. For example, `.stars--0 .stars__star:nth-child(-n + 0)` will not activate any stars because `-n + 0` results in no matches. You should adjust these selectors to match the desired number of active stars. For instance, `.stars--0 .stars__star:nth-child(-n + 0)` should be `.stars--0 .stars__star:nth-child(-n + 1)` to activate the first star, and so on for the other classes.